echo.go

```diff
@@ -169,7 +169,8 @@ const (
 
 // MIME types
 const (
-	MIMEApplicationJSON                  = "application/json"
+	MIMEApplicationJSON = "application/json"
```

Comment:
I did not express correctly. What I meant that it would be helpful to have these rfc link also in doccomments. Something like that 

```go
  // JavaScript Object Notation (JSON) https://www.rfc-editor.org/rfc/rfc8259
	MIMEApplicationJSON = "application/json"
	// Deprecated: use MIMEApplicationJSON instead. JSON should be encoded using UTF-8 by RFC https://www.rfc-editor.org/rfc/rfc8259#section-8.1
	MIMEApplicationJSONCharsetUTF8       = MIMEApplicationJSON + "; " + charsetUTF8
```

Reply:
Ah, I see. I've modified the code according to your suggestion.

How about this?
```go
  // MIMEApplicationJSON JavaScript Object Notation (JSON) https://www.rfc-editor.org/rfc/rfc8259
	MIMEApplicationJSON = "application/json"
	// Deprecated: Please use MIMEApplicationJSON instead. JSON should be encoded using UTF-8.
	// No "charset" parameter is defined for this registration.
	// Adding one really has no effect on compliant recipients.
	// See RFC 8259, section 8.1. https://datatracker.ietf.org/doc/html/rfc8259#section-8.1
	MIMEApplicationJSONCharsetUTF8       = MIMEApplicationJSON + "; " + charsetUTF8
```
https://github.com/labstack/echo/pull/2568/files#diff-aba8ba313c6b989153f1f96e641a8e0e75313485d2e13d64378e70858d5a4669R172-R177

Reply:
@aldas would you mind review my updated PR?